requirements.txt

```diff
@@ -1,6 +1,7 @@
 certifi==2020.12.5
 requests==2.25.1
 watchdog==2.0.3
+infi.systray==0.1.12
```

Comment:
This will need a `; sys_platform == 'win32'` appending to the line, as this is literally only for on windows.

Reply:
Done